README.md

```diff
@@ -42,6 +42,11 @@ Next run the install.sh script. See the i[nstall docs](https://caldera.readthedo
 ```
 ./install.sh
 ```
+Move to your installed python virtualenv withing calendra
+for Mac OS / Linux
```

Comment:
good idea. but we're actually already covering this inside the install.sh. a new virtual environment will be created when you run it, installing the requirements into it automatically.